icons/tools/update-visio.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+ICONS_PATH=${PWD}
+
+GITHUB_ORG="hoveytechllc"
+REPO_NAME="visio-stencil-creator"
+
+rm -fdr ./tools/${REPO_NAME}
+
+# Clone repository in current path
+git clone https://github.com/${GITHUB_ORG}/${REPO_NAME}.git ./tools/${REPO_NAME}
+
+# build image using Dockerfile from github repository
+docker build \
```

Comment:
What if, rather than git clone & build, we pushed the resulting image to k8s.gcr.io and just run from that name?

Reply:
Sure. Either way is fine to me. We can remove `git clone` & `build` all together, and depend on a pre-built image, or modify to `git clone` & `build/push/run` using `k8s.gcr.io/...` tag.